commit 861377730aa9db4cbaa0f3bd3f4d295c152732c4
Author: Tom Lendacky <thomas.lendacky@amd.com>
Date:   Thu Dec 10 11:10:07 2020 -0600

    KVM: SVM: Provide support for SEV-ES vCPU loading
    
    An SEV-ES vCPU requires additional VMCB vCPU load/put requirements. SEV-ES
    hardware will restore certain registers on VMEXIT, but not save them on
    VMRUN (see Table B-3 and Table B-4 of the AMD64 APM Volume 2), so make the
    following changes:
    
    General vCPU load changes:
      - During vCPU loading, perform a VMSAVE to the per-CPU SVM save area and
        save the current values of XCR0, XSS and PKRU to the per-CPU SVM save
        area as these registers will be restored on VMEXIT.
    
    General vCPU put changes:
      - Do not attempt to restore registers that SEV-ES hardware has already
        restored on VMEXIT.
    
    Signed-off-by: Tom Lendacky <thomas.lendacky@amd.com>
    Message-Id: <019390e9cb5e93cd73014fa5a040c17d42588733.1607620209.git.thomas.lendacky@amd.com>
    Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>

diff --git a/arch/x86/kvm/x86.c b/arch/x86/kvm/x86.c
index eea0e9fed62a..3e58612babfe 100644
--- a/arch/x86/kvm/x86.c
+++ b/arch/x86/kvm/x86.c
@@ -197,7 +197,8 @@ EXPORT_SYMBOL_GPL(host_efer);
 bool __read_mostly allow_smaller_maxphyaddr = 0;
 EXPORT_SYMBOL_GPL(allow_smaller_maxphyaddr);
 
-static u64 __read_mostly host_xss;
+u64 __read_mostly host_xss;
+EXPORT_SYMBOL_GPL(host_xss);
 u64 __read_mostly supported_xss;
 EXPORT_SYMBOL_GPL(supported_xss);